prac_07/convert_miles_km.kv

```diff
@@ -0,0 +1,25 @@
+BoxLayout:
+    orientation: 'vertical'
+    BoxLayout:
+        orientation: 'horizontal'
+        TextInput :
+            id : input_number
+            text: '33'
+            on_text: app.calculation()
+        BoxLayout:
+            orientation: 'vertical'
+            Button:
+                text: 'Up'
+                on_press: app.handle_increment(1)
+            Button:
+                text: 'Down'
+                on_press: app.handle_increment(-1)
+#    Button:
+#        text: 'Convert miles to km'
+#        on_press: app.calculation()
+    Label:
+        id:output_number
+        text: '53.10822'
+        color: (1, 1, 0, 1)
```

Comment:
Could this label contain some kind of **_message_** that the python program can talk to?